### PR TITLE
Add guide on filesystem resizing for 1.9

### PR DESCRIPTION
**Pull request by r7vme**
Fixes: https://github.com/giantswarm/giantswarm/issues/3598

### CODE REVIEW

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 1]
```diff
@@ -86,8 +86,66 @@ Starting with Kubernetes v1.9 Persistent Volume Claims can be expanded by simply
 
 In case the volume to be expanded contains a file system, the resizing is only performed when a new Pod is started using the Persistent Volume Claim in `ReadWrite` mode. In other words, if a volume being expanded is used in a Pod or Deployment, you will need to delete and recreate the pod for file system resizing to take place. File system resizing is only supported for XFS, Ext3, and Ext4.
 
+__Warning__: Expanding filesystem is fully supported in Giant Swarm clusters starting from Kubernetes 1.10. Clusters with 1.9 have an issue with `lsblk` utility that is used by Kubernetes to detect filesystem type. See below how to resize filesystem manually in that case.
```
Comment (rossf7) [inline content/guides/using-persistent-volumes-on-aws/index.md]: "the lsblk"  "the filesystem"

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 2]
```diff
@@ -86,8 +86,66 @@ Starting with Kubernetes v1.9 Persistent Volume Claims can be expanded by simply
...
+
 __Warning__: Expanding EBS volumes is a time consuming operation. Also, there is a per-volume quota of one modification every 6 hours.
 
+## Resizing Filesystem For Kubernetes 1.9
+
+This procedure is only required for Kubernetes 1.9. Newer clusters support filesystem resizing "out-of-the-box".
+
+Make sure that EBS volume was successfully resized and PVC has been detached from other pods.
```
Comment (rossf7) [inline content/guides/using-persistent-volumes-on-aws/index.md]: "the EBS volume" "the PVC"

**Comment by tommyhobin** [comment]
I'll keep an eye out and get this posted as an UPDATE either this afternoon or schedule tomorrow a.m., depending on what's going on today in social (haven't looked yet).

Q: May I call the update to the doc "Filesystem resizing for 1.9"(?)

**Comment by r7vme** [comment]
Thanks for review. @rossf7 addressed your comments.

> Q: May I call the update to the doc "Filesystem resizing for 1.9"(?)

Yes, that's correct. :)

**Pull request merged**